INSTALL.md

```diff
@@ -18,12 +18,14 @@
 | 3.6 or later    | python3            | python3            | python3        |
 | 0.18 or later   | gettext-devel      | gettext-tools      | gettext        |
 | 0.18 or later   |                    |                    | autopoint      |
-| 2.12.0 or later | gnutls-devel       | libgnutls-devel    | libgnutls-dev  |
+| 3.5.18 or later | gnutls-devel       | libgnutls-devel    | libgnutls-dev  |
 
 Also:
 * make must be GNU (or compatible) (setting MAKE=gmake might also work but is
   untested)
 * GNU (or compatible) getopt must be somewhere on the PATH
+* gnutls >= 3.6.2 is preferred.  Pacemaker can be built with versions back to
+  3.5.18 but will not be useable in FIPS mode.
```

Comment:
Actually 2.12.0 or later is still accurate for the minimum. Versions 3.3.0 to 3.6.1 will not be usable in FIPS mode.